dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/Contentlet.java

```diff
@@ -219,36 +221,6 @@
     private boolean bool24;
     private boolean bool25;
 
-    //private String folder;
-
-    private File binary1;
-    private File binary2;
-    private File binary3;
-    private File binary4;
-    private File binary5;
-    private File binary6;
-    private File binary7;
-    private File binary8;
-    private File binary9;
-    private File binary10;
-    private File binary11;
-    private File binary12;
-    private File binary13;
-    private File binary14;
-    private File binary15;
-    private File binary16;
-    private File binary17;
-    private File binary18;
-    private File binary19;
-    private File binary20;
-    private File binary21;
-    private File binary22;
-    private File binary23;
-    private File binary24;
-    private File binary25;

```

Comment:
why all this are remove?

Reply:
That code is marked by IntelliJ as not being used by any other class in the source code. It was probable caused by another refactoring not related to this issue.